src/main/java/com/world/entertainment/worldentertainment/service/CountryService.java

```diff
@@ -0,0 +1,73 @@
+package com.world.entertainment.worldentertainment.service;
+
+import com.world.entertainment.worldentertainment.dto.controller.CountryResponse;
+import com.world.entertainment.worldentertainment.dto.controller.CreateCountry;
+import com.world.entertainment.worldentertainment.dto.controller.UpdateCountry;
+import com.world.entertainment.worldentertainment.entity.Country;
+import com.world.entertainment.worldentertainment.exception.EntityNotFoundException;
+import com.world.entertainment.worldentertainment.repository.CountryRepository;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+@Service
+public class CountryService {
+
+    private final CountryRepository countryRepository;
+    private final ModelMapper modelMapper;
+
+    public CountryService(
+            CountryRepository countryRepository,
+            ModelMapper modelMapper
+    ) {
+        this.countryRepository = countryRepository;
+        this.modelMapper = modelMapper;
+    }
+
+    public CountryResponse getById(int id) {
+        var entity = countryRepository.findById(id).orElseThrow(()
```

Comment:
I know that we didn't do it here, but I guess we need to chose entity where deleteAt is null. If you get stack I will help you with request